tests/e2e/navigation/about.wdio-spec.js

```diff
@@ -0,0 +1,47 @@
+const chai = require('chai');
```

Comment:
Why chai?  WDIO has some extended matchers and then [uses jest](https://webdriver.io/docs/api/expect-webdriverio/#default-matchers) for everything else?

Reply:
I prefer chai because it's a strong interface that we already use everywhere, that means that we only need to learn one matcher library, and if we change frameworks we don't need to migrate (like protractor tests to mocha).

Reply:
Honestly, I'm cool with that. We should take the time to do [this ](https://webdriver.io/docs/assertion/#global) so we are not having to require it everywhere and then everyone is just using it out of the box. Or if that's not ideal then we need to let everyone know that Chai is the expected assertion framework and not buried in a resolved PR comment.

Reply:
Cool. I pushed a change to add that and also maybe fixed all tests? TBD :D